### PR TITLE
#155 - dist-tags add command

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #155 
Added IT and slice to handle `dist-tag add` command

### CODE REVIEW

File: src/main/java/com/artipie/npm/http/NpmSlice.java [hunk 1]
```diff
@@ -93,6 +93,17 @@ public NpmSlice(
         final Permissions perms,
         final Authentication auth) {
         this.route = new SliceRoute(
+            new RtRulePath(
+                new RtRule.All(
+                    new ByMethodsRule(RqMethod.PUT),
+                    new RtRule.ByPath(".*/dist-tags/.*")
```
Comment (olegmoz) [inline src/main/java/com/artipie/npm/http/NpmSlice.java:99]: @olenagerasimova I'd suggest to use same pattern for routing and request parsing (e.g. reference `AddDistTagsSlice.PTRN`)

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#160` is not assigned, can't get performer

**Pull request merged**